app/views/insured/plan_shoppings/show.html.slim

```diff
@@ -8,7 +8,10 @@
       .col-lg-8.col-md-8.col-sm-8.col-xs-12
         .ct-1
           h1.heading-text = l10n("insured.plan_shoppings.show.title")
-          h4.switch-header = l10n("insured.plan_shoppings.show.title.content")
+          - if @coverage_kind == "health"
```

Comment:
@jacobkagon Is there a specific reason for using `@coverage_kind == "health"` instead of `@hbx_enrollment.is_health_enrollment?`?

Reply:
@coverage_kind is a static value and this value is already fetched in show action. @hbx_enrollment.is_health_enrollment? is a method call. I think using @coverage_kind gives the upper hand to UI page.